openlibrary/i18n/es/messages.po

```diff
@@ -130,2246 +103,3578 @@ msgstr "Debe tener entre 3 y 20 caracteres"
 
-#: create.html:44 forms.py:50
+#: forms.py:51
 msgid "Choose a screen name"
 msgstr "Escoge un nombre a mostrar"
 
-#: forms.py:52
-msgid "Only letters and numbers, please, and at least 3 characters."
+#: forms.py:53
+msgid "Letters and numbers only please, and at least 3 characters."
 msgstr "Solo letras y números, por favor, y al menos 3 caracteres."
 
-#: forms.py:54 forms.py:104
+#: forms.py:56 forms.py:98
 msgid "Choose a password"
 msgstr "Escoge una contraseña"
 
-#: forms.py:57
+#: forms.py:59
 msgid "Confirm password"
 msgstr "Confirmar contraseña"
 
-#: forms.py:59
+#: forms.py:61
 msgid "Passwords didn't match."
 msgstr "Las contraseñas no coinciden."
 
-#: forms.py:88
-msgid "Invalid password"
-msgstr "Contraseña inválida"
+#: forms.py:62
+msgid "Send me a monthly newsletter from the <a href=\"https://archive.org/\">Internet Archive</a>, the non-profit that runs Open Library"
+msgstr "Envíame un boletín mensual de <a href=\"https://archive.org/\">Internet Archive</a>, la organización sin fines de lucro que administra Open Library"
 
 #: forms.py:91
-msgid "Current password"
-msgstr "Contraseña actual"
-
-#: forms.py:92
-msgid "Choose a new password"
-msgstr "Escoge una nueva contraseña"
-
-#: email.html:24 forms.py:96
-msgid "Your new email address"
-msgstr "Tu dirección de correo electrónico"
+msgid "Invalid password"
+msgstr "Contraseña inválida"
 
-#: edit.html:20
+#: edit.html:13
 msgid "Edit Author"
 msgstr "Editar autor"
 
-#: edit.html:33
+#: edit.html:26
 msgid "Name"
 msgstr "Nombre"
 
-#: edit.html:35
-msgid ""
-"Please use natural order. For example: <strong>Leo Tolstoy</strong> not "
-"<strong>Tolstoy, Leo</strong>."
-msgstr ""
-"Por favor, usa el orden natural. Por ejemplo: <strong>Leo Tolstoy</strong>, no "
-"<strong>Tolstoy, Leo</strong>."
+#: edit.html:28
+msgid "Please use natural order. For example: <strong>Leo Tolstoy</strong> not <strong>Tolstoy, Leo</strong>."
+msgstr "Por favor, usa el orden natural. Por ejemplo: <strong>Leo Tolstoy</strong>, no <strong>Tolstoy, Leo</strong>."
 
-#: edit.html:36 edit.html:153 edit/addfield.html:96 edit/addfield.html:141
+#: edit.html:29 edit.html:92 edit/addfield.html:100 edit/addfield.html:145
 msgid "Required field"
 msgstr "Campo obligatorio"
 
-#: edit.html:50
+#: edit.html:43
 msgid "About"
 msgstr "Acerca de"
 
-#: edit.html:51 edit.html:177
+#: edit.html:44 edit.html:110
 msgid "Links"
 msgstr "Enlaces"
 
-#: edit.html:60
+#: edit.html:53
 msgid "A short bio?"
 msgstr "¿Una breve biografía?"
 
-#: edit.html:61
-msgid ""
-"If you \"borrow\" information from somewhere, please make sure to cite the "
-"source."
-msgstr ""
-"Si tomas «prestada» información de algún otro lugar, por favor, asegúrate de "
-"citar la fuente."
+#: edit.html:54
+msgid "If you \"borrow\" information from somewhere, please make sure to cite the source."
+msgstr "Si tomas «prestada» información de algún otro lugar, por favor, asegúrate de citar la fuente."
```

Comment:
Here is an example of the Latin quotes. I agree that it's better to be consistent, but don't feel qualified to choose which way we standardize? 

@dcapillae What do you think is best?

Reply:
I agree. It's better to be consistent. I always follow the guidelines of the Spanish Language Academy in my translations. The right thing to do is to use Latin quotes according to these guidelines.

However, double quotes are widely used among Spanish speakers and do not conflict with common English quotes, where Latin quotes are not used. So it may be better to use double quotes ("...") in all cases. Although if using Latin quotes does not generate conflicts in the software, we should follow the language guidelines and use Latin quotes.

If it does not generate conflicts in the software, let's use Latin quotes («...»).